### PR TITLE
fix: pass ctx into ticket auth function so can enrich logging

### PR DESCRIPTION
**Pull request by danielgent**

### CODE REVIEW

File: src/authentication-flows/ticket.ts [hunk 1]
```diff
@@ -19,13 +21,15 @@ function getProviderFromRealm(realm: string) {
 }
 
 export async function ticketAuth(
-  env: Env,
+  ctx: Context<{ Bindings: Env; Variables: Var }>,
```
Comment (danielgent) [inline src/authentication-flows/ticket.ts:24]: we're doing this in other helper functions. I don't see the harm

Comment (markusahlstrand) [reply src/authentication-flows/ticket.ts:24]: Have been going both ways here. I'm sure it's a anti-pattern to pass around the ctx, but it's pretty handy.

**Review by markusahlstrand — APPROVED** [review]
LGTM

File: src/authentication-flows/ticket.ts (hunk 1 above)
Comment (danielgent) [reply src/authentication-flows/ticket.ts:24]: Antipattern how? For testing we'd need to create much larger mocks that's for sure... And it seems cleaner to only pass what is required


BUT should we pop off the env vat setting function? And then we'd need to type it?

Comment (danielgent) [reply src/authentication-flows/ticket.ts:24]: Antipattern how? For testing we'd need to create much larger mocks that's for sure... And it seems cleaner to only pass what is required


BUT should we pop off the env vat setting function? And then we'd need to type it?

**Pull request merged**